src/cargo/sources/git/oxide.rs

```diff
@@ -285,3 +295,36 @@ fn cargo_config_to_gitoxide_overrides(config: &Config) -> CargoResult<Vec<String
 
     Ok(values)
 }
+
+// TODO: use `gitoxide` for initialization, understand the `tmp`-dir technique.
```

Comment:
How do we proceed with this TODO?

Reply:
Indeed, let's avoid TODOs and instead trust that it will come up again. It's removed and described in 090666a809f695d0359ef55c17721cc09809a874 .